### PR TITLE
rg-100: redirect to sign in

### PR DESCRIPTION
**Pull request by ValentinaTenus**
add link for redirect from sing up page to log in

### CODE REVIEW

**Review by MariaPetrukhnova — APPROVED** [review]
I've done the same for sig in and sign up))

File: frontend/src/bundles/auth/components/sign-up-form/sign-up-form.tsx [hunk 1]
```diff
@@ -47,7 +48,10 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }) => {
                 <h1 className={styles.registration__title}>Sign Up</h1>
                 <p className={styles.registration__message}>
                     Already have an account? Go to
-                    <span className={styles.registration__link}> Log in</span>
+                    <Link to={'/log-in'} className={styles.registration__link}>
+                        {' '}
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/auth/components/sign-up-form/sign-up-form.tsx:52]: Why do you need it?

File: frontend/src/bundles/auth/components/sign-up-form/sign-up-form.tsx [hunk 2]
```diff
@@ -47,7 +48,10 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }) => {
                 <h1 className={styles.registration__title}>Sign Up</h1>
                 <p className={styles.registration__message}>
                     Already have an account? Go to
-                    <span className={styles.registration__link}> Log in</span>
+                    <Link to={'/log-in'} className={styles.registration__link}>
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/auth/components/sign-up-form/sign-up-form.tsx:51]:
```suggestion
                    <Link to={AppRoute.LOG_IN} className={styles.registration__link}>
```

File: frontend/src/bundles/auth/components/sign-up-form/sign-up-form.tsx (hunk 1 above)
Comment (ValentinaTenus) [reply frontend/src/bundles/auth/components/sign-up-form/sign-up-form.tsx:52]: i don't need it)) only prettify))

**Review by Vitaliy-Kalenichenko — APPROVED** [review]

**Pull request merged**